### DISCUSSION TITLE
List widget

### DISCUSSION THREAD

**Discussion by idansass** [iSoron/uhabits#1600] [Feature Requests]
Hi guys,
Is there really no simple list widget? One where I can see all of today's habits (just like in the app's homescreen), click on a habit's name to open its description, and click on a checkmark next to it to mark it as done?
It seems so basic. Yet there are various, more "complex" widgets which I personally see no need for, and are all along the same line (history, frequency, stats, streaks...). Also the checkmark widget which is strange and seems unfunctional.
I truly like this app so far, but this type of widget is very important as motivation, and as a non-notification constant reminder. Other apps (e.g. HabitNow) have exactly that. But I would really prefer staying with this app, as it has many good qualities.
Please let me know if I'm missing something, or if this basic feature is in development and should be released soon, etc.
Thanks.

**Comment by cijodaniel** [comment]
Hi idansass,
Earlier they had the same feature that you have mentioned in your post, where we can see all the tasks in the home screen and mark it as done 

But I am not sure why they have removed that list widget. Now there are some other widgets which I dont use.

To the team
I appreciate the hardwork done by you guya and would be really happy if you bring back that old list widget feature.

Thank you
